XamarinCommunityToolkit/Behaviors/ConfirmPasswordBehavior.cs

```diff
@@ -0,0 +1,22 @@
+using Xamarin.Forms;
+
+namespace Microsoft.Toolkit.Xamarin.Forms.Behaviors
+{
+    public class ConfirmPasswordBehavior : ValidationBehavior
+    {
+        public static readonly BindableProperty ComparedTextProperty =
+            BindableProperty.Create(nameof(ComparedText), typeof(string),
+                typeof(ConfirmPasswordBehavior), string.Empty);
+        protected override bool Validate(object value)
+        {
+            var confirmPasswordText = value as string;
+            return confirmPasswordText != null && confirmPasswordText == ComparedText;
+        }
+        
+        public string ComparedText
```

Comment:
Since this class has name "ConfirmPasswordBehavior", I believe this property should have the name "OriginalPassowrd" or smth like that? Or probably we should consider renaming this behavior class name "ConfirmEqualInputsBehavior"

@sthewissen @jfversluis @Char0394  thoughts